testing/scripts/seldon_e2e_utils.py

```diff
@@ -49,6 +60,31 @@ def wait_for_shutdown(deployment_name, namespace, timeout="10m"):
     return run(cmd, shell=True)
 
 
+def get_pod_name_for_sdep(sdep_name, namespace, attempts=20, sleep=5):
+    for _ in range(attempts):
+        ret = run(
+            f"kubectl get -n {namespace} pod -l seldon-deployment-id={sdep_name} -o json",
+            shell=True,
+            stdout=subprocess.PIPE,
+        )
+        if ret.returncode == 0:
+            logging.warning(f"Successfully waited for pod for {sdep_name}")
+            break
+        logging.warning(
+            f"Unsuccessful wait command but retrying for SeldonDeployment pod {sdep_name}"
+        )
+        time.sleep(sleep)
+    assert ret.returncode == 0, "Failed to get  pod names: non-zero return code"
+    data = json.loads(ret.stdout)
+    pod_names = []
+    for item in data["items"]:
+        pod_names.append(item["metadata"]["name"])
+    logging.warning(
```

Comment:
Should we change the `warning`s to `info`s?

Reply:
done